src/fr/uca/jgit/model/Folder.java

```diff
@@ -14,7 +11,7 @@
 
 public class Folder implements Node {
     // Mapping Name -> Node
-    private final Map<String, Node> children;
+    private Map<String, Node> children;
```

Comment:
Pourquoi enlever le final ?

Reply:
`this.children = loadFolder(lastLine).children;`
Avec le final, cette instruction dans le changeBranch() ne passera pas. Ou stv, je peux remettre final et faire un boucle pour la suppression puis l'ajout des nouveaux children

Reply:
Ok je comprends alors on l'enlève !